autoload/OmniSharp.vim

```diff
@@ -1,7 +1,12 @@
 let s:save_cpo = &cpo
 set cpo&vim
 
+"Set g:omnisharp_server_type to 'roslyn' or 'v1'
+let g:omnisharp_server_type = 'v1'
 let s:omnisharp_server = join([expand('<sfile>:p:h:h'), 'server', 'OmniSharp', 'bin', 'Debug', 'OmniSharp.exe'], '/')
+let s:omnisharp_roslyn_server = join([expand('<sfile>:p:h:h'), 'omnisharp-roslyn', 'scripts', 'Omnisharp'], '/')
```

Comment:
Is it okay to do this as a separate issue? Would prefer not to maintain versions of the binary at this time.